### PR TITLE
:sparkles: Use space framework in example1

### PR DESCRIPTION
**Pull request by andreyod**
Change example1 to work with the space manager with a KCP space provider
A follow-up PR will add example1-kubeflex that will perform example1 with the space manager and a KubeFlex space provider

## Related issue(s)

Fixes #

### CODE REVIEW

File: docs/content/Coding Milestones/PoC2023q1/example1.md [hunk 1]
```diff
@@ -33,12 +33,24 @@ they appear in this example.
...
+%}
+
 {%
    include-markdown "example1-subs/example1-start-kcp.md"
    start="<!--example1-start-kcp-start-->"
    end="<!--example1-start-kcp-end-->"
 %}
 
+{%
+   include-markdown "example1-subs/example1-kcp-provider.md"
```
Comment (ezrasilvera) [inline docs/content/Coding Milestones/PoC2023q1/example1.md:49]: should it be create-kcp-provider ?

File: docs/content/Coding Milestones/PoC2023q1/example1.md [hunk 2]
```diff
@@ -33,12 +33,24 @@ they appear in this example.
    end="<!--example1-pre-kcp-end-->"
 %}
 
+{%
+   include-markdown "example1-subs/example1-space-manager.md"
```
Comment (ezrasilvera) [inline docs/content/Coding Milestones/PoC2023q1/example1.md:37]: example1-pace-manager-start ?

File: docs/content/Coding Milestones/PoC2023q1/example1.md (hunk 1 above)
Comment (andreyod) [reply docs/content/Coding Milestones/PoC2023q1/example1.md:49]: I can change the name but it seems to long to me

File: docs/content/Coding Milestones/PoC2023q1/example1.md (hunk 2 above)
Comment (andreyod) [reply docs/content/Coding Milestones/PoC2023q1/example1.md:37]: If we do this, the next line will look like: `start="<!--example1-space-manager-start-start-->"`

**Pull request closed without merging**